data/json/furniture.json

```
@@ -2094,6 +2108,7 @@
     "type": "furniture",
     "id": "f_plant_harvest",
     "name": "harvestable plant",
+    "description": "This plant is ready for harvest.",
```

You may want to imply how to harvest it.
```suggestion
    "description": "This plant is ready for harvest.  Examine it more closely to identify how to harvest the plant appropriately.",
```